gapis/server/grpc.go

```diff
@@ -397,6 +397,15 @@ func (s *grpcServer) DCECapture(ctx xctx.Context, req *service.DCECaptureRequest
 	return &service.DCECaptureResponse{Res: &service.DCECaptureResponse_Capture{Capture: capture}}, nil
 }
 
+func (s *grpcServer) GetGraphVisualizationFile(ctx xctx.Context, req *service.GraphVisualizationFileRequest) (*service.GraphVisualizationFileResponse, error) {
+	defer s.inRPC()()
+	graphVisualizationFile , err := s.handler.GetGraphVisualizationFile(s.bindCtx(ctx), req.Capture)
```

Comment:
GetGraphVisualization would probably be better, since nothing is a file yet.

Reply:
Yes. I did the changes.